packages/cf-icons/usage.md

```diff
@@ -24,24 +22,26 @@ Framework.
 
 ## SVG icons
```

Comment:
We could remove this heading and pull everything else up a level, now that the icon font is gone.

Reply:
Done